doc/api.md

```diff
@@ -3,19 +3,51 @@
 There are two ways to manipulate the data in **timeld**.
 
 - The Gateway server HTTP API – import and export of timesheets and projects
-- [**m-ld**](https://m-ld.org/) – live read/write sharing of timesheet content
+- [**m-ld**](https://m-ld.org/) – live read/write sharing of timesheet content ![coming soon](https://img.shields.io/badge/-coming%20soon-red)
 
 ## pre
 
 Data API access requires a _key_, which can be obtained using the Command Line Interface (CLI).
 
 1. [Install and configure](https://www.npmjs.com/package/timeld-cli#install) the CLI.
-2. Open an [administration session](https://www.npmjs.com/package/timeld-cli#admin) and follow the instructions to register the local device. When done, type `exit` ⏎ to leave the admin session.
-3. Use `timeld config` (no options) to display the configuration. Your key is found here: `"ably" : { "key": "{your.key}"`
+2. Open an [administration session](https://www.npmjs.com/package/timeld-cli#admin) and follow the instructions to register the local device.
+3. Type `key` ⏎ and your API access key will be displayed.
+4. Type `exit` ⏎ to leave the admin session.
 
-## http api
+The key provided is specific to your user account, and has the same access rights as you do.
 
-### timesheet/project export
+## http api for timesheets & projects
+
+### import
+
+This end-point allows you to import any number of projects, timesheets and timesheet entries from another system.
+
+- **Request**
+  ```
+  POST /api/import
+  Authorization: Basic {base_64(user, key)}
+  Content-Type: application/x-ndjson
+  ```
+  The import data should be provided as [new-line delimited JSON (NDJSON)](http://ndjson.org/). Each line is a [JSON-LD](https://json-ld.org/) subject. The JSON-LD context can be obtained from https://timeld.org/context. [JSON Type Definitions](https://jsontypedef.com/) for valid subjects (including full property documentation) can be found at https://timeld.org/jtd.
```

Comment:
Would suggest rearranging the last sentence so that there aren't two hyperlinks immediately following one another; this will make it more readable.